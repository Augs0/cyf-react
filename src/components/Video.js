import React from 'react'

export const Video = () => (
  <div className="section-container section-container-gray">
    <div className="container">
        <div className="row">
            <iframe 
              title="CodeYourFuture graduate Ehmed shares his story"
              width="853"
              height="480"
              src="https://www.youtube.com/embed/LpQUHt2N6rY?rel=0&amp;showinfo=0"
              frameBorder="0"
              allowFullScreen>
            </iframe>
      </div>
    </div>
</div>
)