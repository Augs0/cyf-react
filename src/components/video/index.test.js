import React from 'react';
import { shallow } from 'enzyme';

import Video from './';

describe('Video Component', () => {
  it('renders', () => {
    expect(shallow(<Video />)).toMatchSnapshot();
  });
});
