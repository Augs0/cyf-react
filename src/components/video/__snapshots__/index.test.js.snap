// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video Component renders 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Video />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="container"
>
        <div
                className="row"
        >
                <div
                        className="embed-responsive embed-responsive-16by9"
                >
                        <iframe
                                allowFullScreen={true}
                                className="embed-responsive-item"
                                frameBorder="0"
                                height="480"
                                src="https://www.youtube.com/embed/LpQUHt2N6rY?rel=0&showinfo=0"
                                title="CodeYourFuture graduate Ehmed shares his story"
                                width="853"
                        />
                </div>
        </div>
</div>,
      "className": "section-container section-container-gray",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="row"
>
          <div
                    className="embed-responsive embed-responsive-16by9"
          >
                    <iframe
                              allowFullScreen={true}
                              className="embed-responsive-item"
                              frameBorder="0"
                              height="480"
                              src="https://www.youtube.com/embed/LpQUHt2N6rY?rel=0&showinfo=0"
                              title="CodeYourFuture graduate Ehmed shares his story"
                              width="853"
                    />
          </div>
</div>,
        "className": "container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="embed-responsive embed-responsive-16by9"
>
            <iframe
                        allowFullScreen={true}
                        className="embed-responsive-item"
                        frameBorder="0"
                        height="480"
                        src="https://www.youtube.com/embed/LpQUHt2N6rY?rel=0&showinfo=0"
                        title="CodeYourFuture graduate Ehmed shares his story"
                        width="853"
            />
</div>,
          "className": "row",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": <iframe
              allowFullScreen={true}
              className="embed-responsive-item"
              frameBorder="0"
              height="480"
              src="https://www.youtube.com/embed/LpQUHt2N6rY?rel=0&showinfo=0"
              title="CodeYourFuture graduate Ehmed shares his story"
              width="853"
/>,
            "className": "embed-responsive embed-responsive-16by9",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "allowFullScreen": true,
              "className": "embed-responsive-item",
              "frameBorder": "0",
              "height": "480",
              "src": "https://www.youtube.com/embed/LpQUHt2N6rY?rel=0&showinfo=0",
              "title": "CodeYourFuture graduate Ehmed shares his story",
              "width": "853",
            },
            "ref": null,
            "rendered": null,
            "type": "iframe",
          },
          "type": "div",
        },
        "type": "div",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="container"
>
          <div
                    className="row"
          >
                    <div
                              className="embed-responsive embed-responsive-16by9"
                    >
                              <iframe
                                        allowFullScreen={true}
                                        className="embed-responsive-item"
                                        frameBorder="0"
                                        height="480"
                                        src="https://www.youtube.com/embed/LpQUHt2N6rY?rel=0&showinfo=0"
                                        title="CodeYourFuture graduate Ehmed shares his story"
                                        width="853"
                              />
                    </div>
          </div>
</div>,
        "className": "section-container section-container-gray",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="row"
>
            <div
                        className="embed-responsive embed-responsive-16by9"
            >
                        <iframe
                                    allowFullScreen={true}
                                    className="embed-responsive-item"
                                    frameBorder="0"
                                    height="480"
                                    src="https://www.youtube.com/embed/LpQUHt2N6rY?rel=0&showinfo=0"
                                    title="CodeYourFuture graduate Ehmed shares his story"
                                    width="853"
                        />
            </div>
</div>,
          "className": "container",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="embed-responsive embed-responsive-16by9"
>
              <iframe
                            allowFullScreen={true}
                            className="embed-responsive-item"
                            frameBorder="0"
                            height="480"
                            src="https://www.youtube.com/embed/LpQUHt2N6rY?rel=0&showinfo=0"
                            title="CodeYourFuture graduate Ehmed shares his story"
                            width="853"
              />
</div>,
            "className": "row",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": <iframe
                allowFullScreen={true}
                className="embed-responsive-item"
                frameBorder="0"
                height="480"
                src="https://www.youtube.com/embed/LpQUHt2N6rY?rel=0&showinfo=0"
                title="CodeYourFuture graduate Ehmed shares his story"
                width="853"
/>,
              "className": "embed-responsive embed-responsive-16by9",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "allowFullScreen": true,
                "className": "embed-responsive-item",
                "frameBorder": "0",
                "height": "480",
                "src": "https://www.youtube.com/embed/LpQUHt2N6rY?rel=0&showinfo=0",
                "title": "CodeYourFuture graduate Ehmed shares his story",
                "width": "853",
              },
              "ref": null,
              "rendered": null,
              "type": "iframe",
            },
            "type": "div",
          },
          "type": "div",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
